Description: Build the release shared only.
Origin: vendor
Forwarded: not-needed
Author: Francis ANDRE <zosrothko@orange.fr>

diff --git a/build/rules/global b/build/rules/global
index 33c593a..8443528 100644
--- a/build/rules/global
+++ b/build/rules/global
@@ -225,6 +225,9 @@
 LINKMODE = BOTH
 endif
 
+ifeq ($(LINKMODE),SHARED_RELEASE)
+DEFAULT_TARGET = shared_release
+endif
 ifeq ($(LINKMODE),SHARED)
 DEFAULT_TARGET = all_shared
 endif