Description: Include <pcre.h> when building Poco in unbundled mode.
Origin: vendor
Forwarded: not-needed
Author: Francis ANDRE <zosrothko@orange.fr>

--- a/Foundation/include/Poco/RegularExpression.h
+++ b/Foundation/include/Poco/RegularExpression.h
@@ -26,18 +26,19 @@
 #include "Poco/Foundation.h"
 #include <vector>
 
-
+#ifdef POCO_UNBUNDLED
+#include <pcre.h>
+#else
 //
 // Copy these definitions from pcre.h
 // to avoid pulling in the entire header file
 //
 extern "C"
 {
-	struct real_pcre8_or_16;                 /* declaration; the definition is private  */
 	typedef struct real_pcre8_or_16 pcre;
 	struct pcre_extra;
 }
-
+#endif
 
 namespace Poco {
 