Description: Use multiarch tuple when detecting odbc library path.
Origin: vendor
Forwarded: not-needed
Author: Cristian Greco <cristian@debian.org>

--- a/Data/ODBC/ODBC.make
+++ b/Data/ODBC/ODBC.make
@@ -10,6 +10,9 @@
 ODBCINCDIR = /usr/include
 endif
 
+MULTIARCH_TUPLE = $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+ODBCLIBDIR = /usr/lib/$(MULTIARCH_TUPLE)
+
 ifndef ODBCLIBDIR
 ifeq (0, $(shell test -d /usr/lib/$(OSARCH)-linux-gnu; echo $$?))
 ODBCLIBDIR = /usr/lib/$(OSARCH)-linux-gnu
