Description: Add poco shared libs to LD_LIBRARY_PATH.
Origin: vendor
Forwarded: not-needed
Author: Francis ANDRE <zosrothko@orange.fr>

diff --git a/release/script/mkdoc b/release/script/mkdoc
old mode 100755
new mode 100644
index 40f5b0b..2f1eeff
--- a/release/script/mkdoc
+++ b/release/script/mkdoc
@@ -84,8 +84,16 @@
 	osarch=`basename $archpath`
 fi
 
+if [ ${osname:0:6} = "CYGWIN" ] ; then
+	osname="CYGWIN"
+	export PATH=$tools/lib/$osname/$osarch:$PATH
+fi
+
 export PATH=$tools/PocoDoc/bin/$osname/$osarch:$PATH
 echo PATH=$PATH
 
+export LD_LIBRARY_PATH=$tools/lib/$osname/$osarch:$LD_LIBRARY_PATH
+echo LD_LIBRARY_PATH=$LD_LIBRARY_PATH
+
 echo "Building documentation $release (using $docConfig)"
 mkdocumentation $reltag $relspec -C $docConfig -v $version
 