diff --git a/build/config/Linux b/build/config/Linux
index 40ef8f0..386f714 100644
--- a/build/config/Linux
+++ b/build/config/Linux
@@ -20,7 +20,7 @@ CXX     = ${CROSS_COMPILE}g++
 LINK    = $(CXX)
 LIB     = ${CROSS_COMPILE}ar -cr
 RANLIB  = ${CROSS_COMPILE}ranlib
-SHLIB   = $(CXX) -shared -Wl,-soname,$(notdir $@) -o $@
+SHLIB   = $(CXX) -shared -Wl,-soname,$(notdir $@) $(LDFLAGS) -o $@
 SHLIBLN = $(POCO_BASE)/build/script/shlibln
 STRIP   = ${CROSS_COMPILE}strip
 DEP     = $(POCO_BASE)/build/script/makedepend.gcc 
@@ -38,10 +38,10 @@ SHAREDLIBLINKEXT = .so
 #
 # Compiler and Linker Flags
 #
-CFLAGS          = 
+CFLAGS         += 
 CFLAGS32        = 
 CFLAGS64        = 
-CXXFLAGS        = -Wall -Wno-sign-compare
+CXXFLAGS       += -Wall -Wno-sign-compare $(CPPFLAGS)
 CXXFLAGS32      = 
 CXXFLAGS64      = 
 SHLIBFLAGS      = 
@@ -58,11 +58,11 @@
 STATICOPT_LINK  = -static
 SHAREDOPT_CC    = -fPIC
 SHAREDOPT_CXX   = -fPIC
-SHAREDOPT_LINK  = -Wl,-rpath,$(LIBPATH)
+SHAREDOPT_LINK  = 
 DEBUGOPT_CC     = -g -D_DEBUG
 DEBUGOPT_CXX    = -g -D_DEBUG
 DEBUGOPT_LINK   = -g
-RELEASEOPT_CC   = -O2 -DNDEBUG
-RELEASEOPT_CXX  = -O2 -DNDEBUG
-RELEASEOPT_LINK = -O2
+RELEASEOPT_CC   = -DNDEBUG
+RELEASEOPT_CXX  = -DNDEBUG
+RELEASEOPT_LINK = 
